source/conf.py

```diff
@@ -4072,7 +4072,8 @@ def setup(_: Sphinx):
 html_css_files = [
     "css/mattermost-global.css",
     "css/homepage-v1.css",
-    "css/compass-icons.css"
+    "css/compass-icons.css",
+    "https://cdn.jsdelivr.net/npm/typesense-docsearch-css@0.3.0",
```

Comment:
Probably want to bundle this, instead of serve from cdn, to be consistent with existing assets